changelogs/unreleased/gh-10855-config-get-changes-path.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/config
+
+* `config:get()` no longer changes the passed argument if it's been
+  passed as a table (gh-10855).
```

Comment:
`<schema object>:get()` and `<schema object>:set()` are fixed as well. Shold we reflect it in the changelog entry?

Reply:
Applied. Thank you.

Reply:
Ah, nvm, I've to add the tests on `:set()` too.

Reply:
Applied X2. Thank you for suggestion.

Reply:
I've mentioned it in the changelog. Thank you.